src/webapp/azext_webapp/__init__.py

```diff
@@ -20,6 +20,13 @@ def __init__(self, cli_ctx=None):
     def load_command_table(self, _):
         with self.command_group('webapp') as g:
             g.custom_command('remote-connection create', 'create_tunnel')
+
+        with self.command_group('webapp scan') as g:
+            g.custom_command('start-scan', 'start_scan')
+            g.custom_command('get-scan-result', 'get_scan_result')
+            g.custom_command('track-scan', 'track_scan')
+            g.custom_command('get-all-scan-result', 'get_all_scan_result')
```

Comment:
These command names are wildly inconsistent with the CLI. Recommended names:

```
webapp scan start
webapp scan show (or show-result)
webapp scan track
webapp scan list (or list-result)
```

Reply:
Renamed to : 
webapp scan start
webapp scan show-result
webapp scan track
webapp scan list-result